plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -63,6 +63,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                 return false;
 
             case SUPPORTS_TOPN_PUSHDOWN:
+            case SUPPORTS_DEREFERENCE_PUSHDOWN:
```

Comment:
might be  redundant, provided accumulo doesn't support row type

add `SUPPORTS_ROW_TYPE -> false` instead?